model-engine/model_engine_server/infra/gateways/resources/k8s_endpoint_resource_delegate.py

```diff
@@ -238,7 +238,9 @@ def add_datadog_env_to_main_container(deployment_template: Dict[str, Any]) -> No
         [
             {
                 "name": "DD_TRACE_ENABLED",
-                "value": "false" if CIRCLECI else "true",
+                "value": "false"
```

Comment:
nit: is this related to a different pr?

Reply:
we missed adding this in https://github.com/scaleapi/llm-engine/commit/2603b9b6ecd9cb6809870ca11e9081308f172305, happy to create a separate PR if that's cleaner

Reply:
Thanks! It'd be great if you put this change into a separate follow up pr!